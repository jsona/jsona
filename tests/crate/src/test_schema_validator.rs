snapshot_schema_validator_point!(
    "schema.jsona",
    point_spec,
    "@",
    "@bo",
    "@object",
    "bo",
    "bool",
    "object2.k1"
);
