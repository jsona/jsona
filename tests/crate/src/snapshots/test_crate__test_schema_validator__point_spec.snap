---
source: tests/crate/src/test_schema_validator.rs
expression: "serde_json::to_string_pretty(&output).unwrap()"
---
{
  "@": [
    {
      "type": "object",
      "properties": {
        "@null": {
          "type": "null"
        },
        "@bool": {
          "type": "boolean"
        },
        "@object": {
          "type": "object"
        }
      }
    }
  ],
  "@bo": [
    {
      "type": "object",
      "properties": {
        "@null": {
          "type": "null"
        },
        "@bool": {
          "type": "boolean"
        },
        "@object": {
          "type": "object"
        }
      }
    }
  ],
  "@object": [
    {
      "type": "object",
      "properties": {
        "k1": {
          "type": "string"
        },
        "k2": {
          "type": "string"
        }
      }
    }
  ],
  "bo": [],
  "bool": [
    {
      "type": "boolean",
      "default": true
    }
  ],
  "object2.k1": [
    {
      "type": "string"
    }
  ]
}
